src/components/Logo/StarWarsLogo.js

```diff
@@ -0,0 +1,38 @@
+import * as classes from './style.module.scss';
+
+export const LOGO_COMPONENT_NAME = 'logo-component';
+
+class Logo extends HTMLElement {
+  constructor() {
+    super();
+    this._contents = new DocumentFragment();
+
+    this.imageContainer = Logo.imageContainerBuilder();
+
+    this._contents.appendChild(this.imageContainer);
+  }
+
+  connectedCallback() {
+    this.imageContainer.style.backgroundImage = `url(${this.getAttribute(
+      'data-image-url',
+    )})`;
+    this.appendChild(this._contents);
+  }
+
+  static imageContainerBuilder() {
+    const imageContainer = document.createElement('div');
```

Comment:
I would probably resign from imageContainer all together and just wrapped the image in 'a' element.